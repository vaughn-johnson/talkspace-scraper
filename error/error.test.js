import { errorMessage } from './index.js';

describe('errorMessage', () => {
  const messages = [
    [undefined],
    [null],
    ['error'],
    ['e'.repeat(2 ** 5)],
    ['two differnt\nlines'],
  ];
  test.each(messages)(
    'matches snapshot with message %p',
    (msg) => expect(errorMessage(msg)).toMatchSnapshot(),
  );
});
