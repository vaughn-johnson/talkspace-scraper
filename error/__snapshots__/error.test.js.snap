// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errorMessage matches snapshot with message "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee" 1`] = `
"
========================================
==  eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee  ==
========================================
"
`;

exports[`errorMessage matches snapshot with message "error" 1`] = `
"
=============
==  error  ==
=============
"
`;

exports[`errorMessage matches snapshot with message null 1`] = `
"
============
==  null  ==
============
"
`;

exports[`errorMessage matches snapshot with message undefined 1`] = `
"
=================
==  undefined  ==
=================
"
`;
